EmotiBit.cpp

```
@@ -3196,6 +3223,62 @@ void EmotiBit::readSensors()
 						led.setState(EmotiBitLedController::Led::BLUE, true);
 						led.setState(EmotiBitLedController::Led::YELLOW, true);
 					}
+
+					if (getPowerMode() == PowerMode::BLUETOOTH) {
```

Clang (14.0.6)

[note] +5, including nesting penalty of 4, nesting level increased to 5

(clang)